packages/mdc-top-app-bar/constants.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright 2017 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/** @enum {string} */
+const strings = {
+  NAVIGATION_EVENT: 'MDCTopAppBar:nav',
+  TITLE_SELECTOR: '.mdc-top-app-bar__title',
```

Comment:
Is this used (or will it be)?

Reply:
It will be used in the next PR. It won't hurt to leave it in for now.